Employeemanagement.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace EmployeeManagementt
+{
+    public class Employee
+    {
+        private static int count = 1000;
+        public string Id { get; private set; }
+        public string Name { get; set; }
+        public double Salary { get; set; }
+
+        public Employee(int id)
+        {
+            count++;
+            Id = "Emp" + (1000+id);
+        }
+        public Employee(int id, string name) : this(id)
+        {
+            Name = name;
+        }
+
+        public Employee(int id, string name, double salary) : this(id, name)
+        {
+            Salary = salary;
+        }
+        public void display()
+        {
+            Console.WriteLine(Id + " " + Name + " " + Salary);
+        }
+        public static void displaynumberofemployees()
+        {
+            Console.WriteLine("number of employees " + (count-1000));
+        }
+        internal class Program
+        {
+            static void Main(string[] args)
+            {
+
+                Employee e1 = new Employee(1, "John Doe", 15000);
```

Comment:
employeetype?

Reply:
Employee(int id)   : assign only Id property
Employee(int id, string name) : assign only Name property
Employee(int id, string name, double salary) : assign only Salary property
Create employee object using the third constructor and Display details by calling Display method


as in the question object was created only with the above three.
Now employeeType is also included.